README.md

```diff
@@ -40,6 +40,19 @@ We have chosen the Rust programming language for Bee because of its C/C++ like p
 
 :::
 
+## Development
+
+The Bee repository has different branches:
+
+|Branch|Description|
+|------|-----------|
+|`mainnet`|The latest release for the IOTA protocol|
+|`mainnet-develop*`|Ongoing development for the IOTA protocol|
```

Comment:
why the `*`?

Reply:
Because we have `mainnet-develop` and `mainnet-develop-0.4` right now.